src/JankProfiler.php

```diff
@@ -71,9 +71,14 @@ public function reset()
         $this->wrapped = null;
     }
 
-    public function jsonSerialize($options = 0)
+    public function jsonSerialize()
     {
-        return json_encode($this->calls, $options);
+        return $this->calls;
+    }
+
+    public function toJson($options = 0)
```

Comment:
I think I'd rather not have this method. Since `JankProfiler` implements `JsonSerializable` people can just pass their instance to `json_encode`.